3.1.0/rockcraft.yaml

```diff
@@ -36,8 +32,6 @@ parts:
     source-tag: 3.1.0
     stage-packages:
       - python3.12-venv
```

Comment:
nit: i suspect that `python3-venv` needs to be staged instead of `python3.12-venv`